compio-http/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "compio-http"
+version = "0.1.0"
+categories = ["asynchronous", "network-programming"]
+keywords = ["async", "net"]
+edition = { workspace = true }
+authors = { workspace = true }
+readme = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+
+[dependencies]
+compio-buf = { workspace = true }
+compio-io = { workspace = true, features = ["compat"] }
+compio-runtime = { workspace = true }
+compio-net = { workspace = true, features = ["runtime"] }
+compio-tls = { workspace = true }
+
+native-tls = { workspace = true, optional = true }
+rustls = { workspace = true, optional = true }
+rustls-native-certs = { workspace = true, optional = true }
+
+cfg-if = { workspace = true }
+hyper = { workspace = true }
+send_wrapper = { workspace = true, features = ["futures"] }
+tokio = { workspace = true }
```

Comment:
Why tokio by default

Reply:
It's only for `tokio::io`, which is used by hyper.